flickr.py

```diff
@@ -77,9 +80,9 @@ def __str__(self):
 print("\n= vs = >> \n")
 
 for photo in photos_list:
-    print(photo)
+    #print(photo)
 
     # if you get encoding error, try this
-    # print(str(photo).encode('utf-8'))
+    print(str(photo).encode('utf-8'))
```

Comment:
Yes, yes, yes. Uncomment it all. Free the code

Reply:
This is what you get for using a Windows machine!!!!!!